src/HtmlUtils.js

```diff
@@ -136,6 +142,33 @@ var sanitizeHtmlParams = {
             attribs.rel = 'noopener'; // https://mathiasbynens.github.io/rel-noopener/
             return { tagName: tagName, attribs : attribs };
         },
+        '*': function(tagName, attribs) {
+            // Only allow certain CSS attributes to avoid XSS attacks
+            // Sanitizing values to avoid `url(...)` and `expression(...)` attacks
+            if (!attribs.style) {
+                return { tagName: tagName, attribs: attribs };
+            }
+
+            const pairs = attribs.style.split(';');
+            let sanitisedStyle = "";
+            for (let i = 0; i < pairs.length; i++) {
```

Comment:
>hand-sanitizing CSS like this scares me quite a lot

Should we look into a library that does this instead?

>will fail if there's any white-space in the expression
>defining our own custom derivative of CSS

Agreed that we shouldn't reinvent the wheel here, but the rationale was to be super strict such as not to allow any possibility of `url(...)` or `expression(...)`. This will inevitably end up being are own version of CSS but are we not already doing that with HTML?